components/Timeline.jsx

```diff
@@ -3,7 +3,7 @@ import { TIMELINE_DATA } from "../data/timeline";
 import { UnderlineTypes } from "../utils/underlineType";
 import Heading from "./Heading";
 
-const TIMELINE_CONTAINER = "bg-white flex flex-col";
+const TIMELINE_CONTAINER = "bg-white flex flex-col lg:h-0 lg:min-height-full";
 const PURPLE_SIDE_LINES_DIR = "/svgs/timeline/left_side_lines.svg";
 
 const Timeline = () => {
```

Comment:
Add a little more padding to top of `Timeline` title? Also to the `Executives` title probably

Reply:
I made them both 2.5rem